app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetController.scala

```diff
@@ -153,4 +156,7 @@ class EditSetController @Inject() (
     }
   }
 
+  private def formToEither[A](form: Form[A]): Either[Form[A], A] =
```

Comment:
This would be useful in other places. Should we put it somewhere common now, or wait until another needs it? The other issue with the latter, is that they may not be aware of it.

Reply:
I'm wary of creating "utility" files full of helper methods at this point where they are only used in one place, however as soon as it's used in another place we should put it in a controllers package object, I think.

I think we can just flag it in code review if it come up?

Reply:
OK.